lib/components/app/responsive-webapp.js

```diff
@@ -180,8 +183,12 @@ class RouterWrapper extends Component {
   }
 
   render () {
-    const { routerConfig } = this.props
-    return (
+    const { dispatch, persistence, routerConfig } = this.props
+    // Initialize Auth0Provider in this component so it is available everywhere.
+    const auth0Config = getAuth0Config(persistence)
+    const auth0Callbacks = getAuth0Callbacks(dispatch)
```

Comment:
As mentioned in another comment, these callbacks should be connected to the component as actions.

Reply:
Good point. Very good point.